### PR TITLE
chore: major update dependencies

### PR DESCRIPTION
**Pull request by alex-gilin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
 {
   "name": "@sap-webide/webide-client-tools",
-  "version": "3.1.2",
+  "version": "3.1.3",
```
Comment (bd82) [inline package.json:3]: don't bump version manually, according to the docs the bumping is automated
- https://github.com/SAP/webide-client-tools/blob/master/CONTRIBUTING.md#releasing-new-version

File: test/bundling_spec.js [hunk 2]
```diff
@@ -621,10 +621,12 @@ describe("The, Exported bundling APIs", () => {
           const nestedDistFolder = path.resolve(__dirname, "resources/dist")
 
           it("won't go into endless loop", () =>
-            bundlingApi.bundleFeature(pkg, {
-              bundler,
-              outDir: nestedDistFolder,
-            }))
+            expect(
```
Comment (bd82) [inline test/bundling_spec.js:624]: 👍

**Comment by bd82** [comment]
Try to test functional flows using the example pacakge
- https://github.com/SAP/webide-client-tools/tree/master/example

File: package.json (hunk 1 above)
Comment (alex-gilin) [reply package.json:3]: reverted

**Review by bd82 — APPROVED** [review]
I think **major** version should be released for this.

**Pull request merged**